src/selfie_tools.c

```diff
@@ -684,12 +693,10 @@ int selfie_read_config_file(params_in *in)
 int selfie_read_env_vars(params_in *in)
 {
   char *list[] = ENVVARS_RUNTIME;
-  int nb_vars = 0;
   char *tmp_string = NULL;
 #ifdef HAVE_DEBUG
   PINFO("");
 #endif
-  nb_vars = sizeof(list) / sizeof(list[0]);
```

Comment:
wasn't used, I think original code wanted to loop over the variables but it might be more clear to explicit them and remove the list now...